File: week-03/friday/01.py
# Create a `Circle` class that takes it's radius as constructor parameter
# It should have a `get_circumference` method that returns it's circumference
# It should have a `get_area` method that returns it's area
import math

class Circle(object):

    def __init__(self, radius):
        self.radius = radius

    def get_circumference(self):
        return 2 * math.pi * self.radius

    def get_area(self):
        return self.radius ** 2 * math.pi


circle = Circle(6)
print("Circumference is: %.3f" % (circle.get_circumference()))
print("Area is: %.3f" % (circle.get_area()))

File: week-08/monday/01.js
'use strict';

// Create a constructor for creating Animals.
// it should take one parameter: what the animal says
// Every animal should have a method called say() that prints what the animal says

function Animals(animalSound) {
  this.animalSound = animalSound;
  this.say = function() {
    console.log(this.animalSound);
  };
}

/*Animals.prototype.say = function () {
  console.log(this.animalSound);
};*/

var kecske = new Animals("meeee");
kecske.say();
